CronResponce.php

```diff
@@ -46,9 +47,6 @@
     }
     
     $handle = fopen(_PS_MODULE_DIR_ . 'sendinblue/csv/SyncToSendinblue.csv', 'w');
-    $key_value = array();
-    $key_value[] = '';
-    fputcsv($handle, $key_value, '');
```

Comment:
By removing these lines, you do write anymore in the file. Why do you still open then ?